content/blog/2023/09/camel-quarkus-release-3.2.0/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Camel Quarkus 3.2.0 Released"
+date: 2023-09-15
+authors: ["zbendhiba"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 3.2.0 Released"
+summary: "Camel Quarkus 3.2.0 is the first final 3.x major release"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the 3.2.0 release of Camel Quarkus. This is the first major 3.x release.
+It aligns with Quarkus 3.2.5.Final and Camel 4.0.0.
+
+## Camel 4.0.0-M3
+Please refer to the [Camel 4.0.0 announcement](/blog/2023/08/RELEASE-4.0.0/) for more details.
+
+## Quarkus 3.2.5.Final
```

Comment:
Also here, should not be 3.2.6 final?

Reply:
Yeah I don't know how to explain this. yeah for end user, maybe 3.2.6.Final, but I didn't release anything based on this version.

Reply:
we can add a sentence to explain that However, Camel Quarkus in the Quarkus platform since Quarkus 3.2.6.Final ?

Reply:
True. I think we can say that the Camel Quarkus has been released with 3.2.5 platform and then the user who wants to leverage Camel Quarkus via Quarkus Platform BOMs mechanism has to use the platform with version 3.2.6 instead.